examples/src/main/java/ExampleDownloadRecording.java

```diff
@@ -0,0 +1,38 @@
+import com.messagebird.MessageBirdClient;
+import com.messagebird.MessageBirdService;
+import com.messagebird.MessageBirdServiceImpl;
+import com.messagebird.exceptions.GeneralException;
+import com.messagebird.exceptions.NotFoundException;
+import com.messagebird.exceptions.UnauthorizedException;
+
+public class ExampleDownloadRecording {
+    public static void main(String[] args) {
+        if (args.length < 5) {
```

Comment:
I need values to test examples in my local. need help ;)

Reply:
I send them from slack

Reply:
working successfully thanks